subprojects/distributions/distributions.gradle

```diff
@@ -90,6 +84,7 @@ task allZip(type: Zip) {
 }
 
 task binZip(type: Zip) {
+    outputs.cacheIf { true }
```

Comment:
We could - but we actually don't want to. When the no timestamped version change is done the `bin` and `all` distribution will change with every commit - since they contain time stamps. We are actually only caching `binZip` since we cannot yet express that we only depend on its contents and not on the zip file itself. We could also say that `binZip` is actually the same as `intTestImage` and drop the input alltogether. WDYT?